diff --git a/node_modules/@react-navigation/material-bottom-tabs/src/views/MaterialBottomTabView.tsx b/node_modules/@react-navigation/material-bottom-tabs/src/views/MaterialBottomTabView.tsx
index 1de5fc3..3cb3c85 100644
--- a/node_modules/@react-navigation/material-bottom-tabs/src/views/MaterialBottomTabView.tsx
+++ b/node_modules/@react-navigation/material-bottom-tabs/src/views/MaterialBottomTabView.tsx
@@ -122,7 +122,7 @@ function MaterialBottomTabViewInner({
   return (
     <BottomNavigation
       {...rest}
-      theme={theme}
+      theme={rest.theme ? { ...theme, ...rest.theme } : theme}
       navigationState={state}
       onIndexChange={(index: number) =>
         navigation.dispatch({
